dbt/models/vcerare/out_vcerare__hourly_available_capacity_factor/schema.yml

```diff
@@ -14,41 +14,54 @@ sources:
                 - datetime_utc
         columns:
           - name: state
+            data_tests:
+              - not_null
           - name: county_or_lake_name
             data_tests:
               - not_null
               - dbt_expectations.expect_column_values_to_not_be_in_set:
                   value_set:
                     - bedford_city
                     - clifton_forge_city
+                    - lake_hurron
+                    - lake_st_clair
           - name: datetime_utc
             data_tests:
               - not_null
               - dbt_expectations.expect_column_values_to_not_be_in_set:
                   value_set:
                     - "{{ dbt_date.date(2020, 12, 31) }}"
           - name: report_year
+            data_tests:
+              - not_null
           - name: hour_of_year
             data_tests:
               - not_null
               - dbt_expectations.expect_column_max_to_be_between:
-                  min_value: 8759
-                  max_value: 8761
+                  min_value: 8760
+                  max_value: 8760
```

Comment:
We always expect the max to == 8760 and by default this test includes the upper and lower bounds. (>=min and <=max)